tests/testthat/testExampleConfig.R

```diff
@@ -27,16 +27,17 @@ test_that("test return value and outputs", {
   # --------- MAKE EXPECTATIONS --------------------
   
   # check format of return value
-  expect_length(processedData, 4)
-  expect_length(processedData$memoryCorrected, 3)
-  expect_length(processedData$calibrated, 3)
-  expect_length(processedData$processed, 3)
-  expect_length(processedData$pooledStdDev, 3)
+  expect_true(is.list(processedData))
+  expect_length(processedData, 3)
+  #expect_length(processedData$memoryCorrected, 3)
```

Comment:
let's add a TODO comment. `# TODO: uncomment these tests when the new interface is stable`. That way we can find this section when searching for "TODO".

Reply:
Good idea; I commented out the failing tests and added TODO flags: 47f226b728475513ad5c5977cc1b9c6720874f1c.